Otec-iOS/Otec/LoginViewController.swift

```diff
@@ -13,63 +13,41 @@ class LoginViewController: UIViewController {
-//                             "newspaper_name":"Daily Planet",
-//                             "created_at":"19-12-1989"]
-//        
-//        guard let new = News.init(dictionary:newDictionary) else {
-//            print ("failure creating new")
-//            return
-//        }
-//        let future3 = NewsRepository().findAll()
-        
-        
         
-//        let future2 = NewsRepository().findAll()
-        
-        // Do any additional setup after loading the view.
+        self.checkIfLoggedInThenShowFeed()
     }
 
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+    private func checkIfLoggedInThenShowFeed() {
+        if (NSUserDefaults.standardUserDefaults().objectForKey(Constants.newspaperUserDefaultsKey) != nil) {
+            self.pushFeedsViewController(animated: false)
+        }
+    }
+    
+    private func pushFeedsViewController(animated animated: Bool) {
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let viewController = storyboard.instantiateViewControllerWithIdentifier("NewsFeedViewController") as! NewsFeedViewController
+        self.navigationController?.pushViewController(viewController, animated: animated)
     }
     
     @IBAction func login(sender: AnyObject) {
-        let newspaperDictionary = ["name":newspaperTextfield.text!,
-                                   "description":"El diario de Clark Kent"]
+        let newspaperName = newspaperTextfield.text!
         
-        let newspaper = Newspaper.init(dictionary:newspaperDictionary)
-        let future = NewspaperRepository().findByID(newspaper!.id)
+        let future = NewspaperRepository().findByID(newspaperName)
         future.start() { result in
             switch result {
-            case .Success(let new):
-                print ("newspaper \(new.id)")
-            // You've got your users fetched in this array!
+            case .Success(let newspaper):
+                dispatch_async(dispatch_get_main_queue()) {
+                    NSUserDefaults.standardUserDefaults().setObject(newspaper.id, forKey:Constants.newspaperUserDefaultsKey)
+                    self.pushFeedsViewController(animated: true)
+                }
             case .Failure(let error):
                 print ("error : \(error)")
-                // You've got a discrete ServerBackendError indicating what happened
             }
         }
     }
 
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
-        // Get the new view controller using segue.destinationViewController.
-        // Pass the selected object to the new view controller.
+    @IBAction func loginAsGuestUser(sender: AnyObject) {
```

Comment:
Unless there are multiple kind of senders, we should specify its type and prevent it from being `AnyObject`.